features/polyExplorer/src/components/buttons/linkButton/linkButton.jsx

```diff
@@ -14,7 +14,7 @@ const LinkButton = ({
     const history = useHistory();
     let changedNavigationState = navigationState;
 
-    const onClickButton = () => {
+    const onClickButton = async () => {
```

Comment:
That change looks unnecessary to me - or am I missing something?

Reply:
Absolutely, tried doing stuff async